src/functions/addProvisionalTechRecord.ts

```diff
@@ -20,6 +20,7 @@ const addProvisionalTechRecord = async (event: any) => {
   if (!msUserDetails || !msUserDetails.msUser || !msUserDetails.msOid) {
     return Promise.resolve(new HTTPResponse(400, formatErrorMessage("Microsoft user details not provided")));
   }
+  delete techRec[0].historicVin;
```

Comment:
Will this throw an error if the historicVin attribute isn't defined? Would it be better to have `delete techRec[0]?.historicVin`

Reply:
It won't throw an error if historicVin is undefined. It will throw an error if techRec[0] is undefined, but that is checked further up.

Reply:
Why is historicVin being removed?

Reply:
Presumably we don't want historicVin adding to the record in dynamodb, it acts as metadata. But I don't think the API client should be sending historicVin in the body payload anyway if that is why we're deleting it here?

Reply:
It's not great that we have calculated fields on the API. Perhaps best to keep the logic here rather than in the clients.